### PR TITLE
Allow hints to be placed on options

### PR DESCRIPTION
**Pull request by andymoody**
Add hints to radio and checkbox option values. E.G. screenshot

### CODE REVIEW

File: views/partials/forms/option-group.html [hunk 1]
```diff
@@ -15,11 +15,12 @@
                 required="true"
                 {{#toggle}} data-toggle="{{toggle}}"{{/toggle}}
                 {{#selected}} checked="checked"{{/selected}}
-                {{^error}}{{#hint}} aria-describedby="{{key}}-hint"{{/hint}}{{/error}}
+                {{^error}}{{#hint}} aria-describedby="{{key}}-{{value}}-hint"{{/hint}}{{/error}}
```
Comment (daniel-ac-martin) [inline views/partials/forms/option-group.html:18]: This currently references a hint above. Are you saying this is a bug?

**Review by daniel-ac-martin — DISMISSED** [review]
It's not clear what the purpose of this change is.

Could a description please be added with a rationale for why this is being done?

A screenshot or two would also probably help.

File: views/partials/forms/option-group.html (hunk 1 above)
Comment (andymoody) [reply views/partials/forms/option-group.html:18]: I would think so - the data passed to the template in the 'hint' variable in this context comes from the option itself, not from the main field. If there is a hint present it should be pointing to the hint for the option (not previously present)

**Comment by daniel-ac-martin** [comment]
_If I understand this correctly_, then this change is correct in that it brings the code into closer alignment to the current GovUK Design System. (See: https://design-system.service.gov.uk/components/radios/#radio-items-with-hints)

However it doesn't go far enough as it leaves the existing hint unreferenced. What is missing is `aria-describedby="{{key}}-hint"` on the `fieldset` element.

**Comment by daniel-ac-martin** [comment]
In terms of the version this strikes me as a new feature. (There were no hints on options before.)

In that case the version should be `4.4.0`.

What do others think?

**Comment by andymoody** [comment]
I have fixed the aria-describedBy and amended the versioning accordingly

**Review by daniel-ac-martin — COMMENTED** [review]
This LGTM but I'd like to know what others think.

**Review by sulthan-ahmed — APPROVED** [review]
Lgtm

**Comment by sulthan-ahmed** [comment]
So i believe from what i remember hof theme allows backward compatibility. When hof was tested using the then New gds designs, there were a lot of breaking changes to apps using old hof. Therefore, to prevent this we only added the new styles into hof-theme and updated the mixins. It means, the old hof apps could continue updating other dependencies without breaking. Anyone that wanted the new gds designs, would add this package & update

**Pull request merged**